rpython/jit/metainterp/test/test_float.py

```diff
@@ -84,6 +84,18 @@ def g(i):
         res = self.interp_operations(g, [-12345])
         assert type(res) is float and res == float(long(r_uint(-12345)))
 
+    def test_max(self):
+        def f(a, b):
+            return max(a, b)
+        res = self.interp_operations(f, [-5.25, 3])
+        assert res == 3
+        # TODO(emacs): Figure out why nan causes issues here
```

Comment:
Would it be because "If only one value is a NaN (SNaN or QNaN) for this instruction, the second source operand, either a NaN or a valid floating-point value, is written to the result"?

https://www.felixcloutier.com/x86/maxsd